experimental/dds/tree2/src/feature-libraries/editable-tree/editableField.ts

```diff
@@ -78,6 +78,9 @@ export function makeField(
 			targetSequence.free();
 		});
 	}
+	context.forest.anchors.on("treeChanging", () => {
```

Comment:
If anchorSet is going to store the generation number, it should keep it updated itself without depending on having an editable tree do it: Its perfectly reasonable to have an anchor-set without an editable tree. I think the entire logic should just provide a faster way to getDetachedFieldContainingPath on anchorNodes, implemented in anchorset (and getDetachedFieldContainingPath  can test for an anchornode and fast path it when walking the path)

Reply:
Updated `anchorSet` to increment its own `generationNumber`.